react/AdminExample.tsx

```diff
@@ -113,6 +138,38 @@ function AdminExample() {
           currency: 'USD',
         },
       },
+      {
+        id: 'menu',
+        resolver: {
+          type: 'root',
+          render: function Render(_props) {
+            const state = useMenuState({})
+
+            return (
+              <Menu
+                state={state}
+                onClick={(
+                  event: React.MouseEvent<HTMLDivElement, MouseEvent>
+                ) => event.stopPropagation()}
+              >
+                <MenuButton
+                  display="actions"
+                  variant="adaptative-dark"
+                  aria-label="Necessary aria-label"
```

Comment:
Should aria labels be translated as well?

Reply:
Yes, the `aria-label` is read by the screen reader